### DISCUSSION TITLE
offer dropdown interface for integer type

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#5821] [Feature Requests]
similar in concept to an enum.

it already works if you manually put `select-dropdown` as the `interface` in `directus_fields`. Even the ui to add the choices works as expected. Only the choices values are strings and not integers and that causes the ui to not recognize the selected value in the dropdown (inserting/updating works, tho). Again, this can be fixed manually just changing the `value` json fields in the `options` column in `directus_fields` to actual integers instead of strings

**Comment by azrikahar** [comment]
Resolved by #10274
